api/repositories/route_repository.go

```diff
@@ -29,7 +29,8 @@ type DestinationRecord struct {
 	AppGUID     string
 	ProcessType string
 	Port        int
-	// Weight and Protocol intentionally omitted as experimental features
+	Protocol    string
```

Comment:
When we fetch a route back out how is this set to `http1`? I don't see it on the CRD definition:
https://github.com/cloudfoundry/cf-k8s-controllers/blob/3e602c6042ab48fb01853e7089ab114de80b7396/controllers/apis/networking/v1alpha1/cfroute_types.go#L33

Is this just being presented as `http1` somewhere?